test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -100,162 +97,161 @@ describe('<AddressSection>', () => {
+    expect(tree.find('select')).to.have.lengthOf(0);
 
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
+    expect(tree.find('select')).to.have.lengthOf(2);
 
-    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'select')).to.not.be.empty;
-
-    tree.click('button.usa-button-outline');
-
-    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'select')).to.be.empty;
+    // Click the cancel button
+    tree.find('button.usa-button-outline').simulate('click');
+    expect(tree.find('select')).to.have.lengthOf(0);
   });
 
   // NOTE: This is a bit of a misnomer; it only tests if countries are unavailable, but that should be sufficient
   it('should show addressUpdateUnavailable if countries or states lists aren\'t available', () => {
     const props = Object.assign({}, defaultProps, { countriesAvailable: false });
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...props}/>);
-    const tree = getFormDOM(component);
+    const tree = mount(<AddressSection {...props}/>);
 
     // Start editing
-    tree.click('button.usa-button-outline');
-    expect(tree.getElement('.usa-alert-heading').textContent).to.contain('Address update unavailable');
+    tree.find('button.usa-button-outline').simulate('click');
+    expect(tree.find('.usa-alert-heading').text()).to.contain('Address update unavailable');
   });
 
   it('should load address in new props after mounting', () => {
     const props = cloneDeep(defaultProps);
     props.savedAddress = {};
-    const tree = SkinDeep.shallowRender(<AddressSection {...props}/>);
+    const tree = mount(<AddressSection {...props}/>);
 
-    const instance = tree.getMountedInstance();
-    expect(instance.state.editableAddress).to.equal(props.savedAddress);
+    expect(tree.state('editableAddress')).to.equal(props.savedAddress);
+    tree.find('.usa-button-outline').simulate('click');
 
-    const newProps = Object.assign({}, props, { savedAddress: { addressOne: '123 Main St' } });
-    instance.componentWillReceiveProps(newProps);
-    expect(instance.state.editableAddress).to.equal(newProps.savedAddress);
+    // Edit the street address
+    const newAddress = '123 Main St';
+    tree.find('input[name="addressOne"]').simulate('change', { target: { value: newAddress } });
+    expect(tree.state('editableAddress').addressOne).to.equal(newAddress);
   });
 
   it('should not call saveAddress when Cancel is clicked', () => {
     saveSpy.reset();
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...defaultProps}/>);
-    const tree = getFormDOM(component);
+    const tree = mount(<AddressSection {...defaultProps}/>);
 
     // Start editing
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
 
     // Try to save
-    tree.click('button.usa-button-outline');
+    tree.find('button.usa-button-outline').simulate('click');
```

Comment:
This isn't your change but this second button click actually clicks the 'cancel update' button in the address editor, so might be nice to update the corresponding comment